src/scripts/events/navigationEvents.js

```diff
@@ -23,7 +25,17 @@ const navigationEvents = () => {
   // 2. Convert the response to an array because that is what the makeAuthors function is expecting
   // 3. If the array is empty because there are no authors, make sure to use the emptyAuthor function
   document.querySelector('#authors').addEventListener('click', () => {
-    console.warn('CLICKED AUTHORS');
+    getAuthors().then((authorsArray) => {
+      if (authorsArray.length !== 0) {
```

Comment:
```suggestion
      if (authorsArray.length) {
```
If this results in 0, it is falsy, so then it would result in the else statement running. So the `!== 0` is not needed